addons/website/static/src/js/tours/tour_utils.js

```diff
@@ -98,11 +104,11 @@ export function changeColumnSize(position = "right") {
 export function changeImage(snippet, position = "bottom") {
     return [
         {
-            trigger: "body.editor_enable",
+            trigger: "body :iframe .odoo-editor-editable",
```

Comment:
previously editor_enable class was added when we have SelectMenu, maybe we should check ```.o_builder_sidebar_open```